Day2/Dictionaries.md

```diff
@@ -2,14 +2,14 @@
 |----|----|----|
 # Dictionaries
 
-In Python, a dictionary is an unordered collection of elements indexed by a non-numeric key.  A dictionary can be thought of as a list where the index is a string rather than a number, e.g. myDict["bread"] rather than myList[2].
+In Python, a dictionary is an unordered collection of elements indexed by a non-numeric key.  A dictionary can be thought of as a list where the index is a string rather than a number, e.g. groceryDict["bread"] rather than myList[2].
```

Comment:
Python dictionaries can be indexed by numbers also
'''
a = {}
a["a"] = "A"
a[65] = "A"
print a        # (Python 2.7) will display {'a': 'A', 65: 'A'}
'''